pandas/io/parsers/readers.py

```diff
@@ -674,6 +674,14 @@ def _read(
     # Extract some of the arguments (pass chunksize on).
     iterator = kwds.get("iterator", False)
     chunksize = kwds.get("chunksize", None)
+
+    # Check type of encoding_errors
+    errors = kwds.get("encoding_errors", "strict")
+    if not isinstance(errors, str) and errors is not None:
+        raise ValueError(
+            f"encoding_errors must be a string or None, got {type(errors).__name__}"
```

Comment:
```suggestion
            f"encoding_errors must be a string, got {type(errors).__name__}"
```

Reply:
Hi, Roeschke. I think None should be considered because it can't pass the check when encoding_errors = None. 
Actually, I didn't add 'is not None' the first time but the check was failed.

Reply:
Which check?

Reply:
I commit your suggestion now, so I can find which check.

Reply:
> Which check?

ValueError: encoding_errors must be a string , got NoneType

Reply:
That test just needs to be updated to account for this new error message